languagetool-language-modules/de/src/main/resources/org/languagetool/rules/de/grammar.xml

```diff
@@ -83767,6 +83768,17 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
             </rule>
         </rulegroup>
         <rulegroup id="DE_MULTITOKEN_SPELLING_TWO" name="Rechtschreibfehler in Eigennamen (2 Token)">
+            <antipattern>
+                  <token postag="UNKNOWN"/>
+                  <token postag="UNKNOWN" regexp="yes">.+ies</token>
+                  <example>Wir mögen alle Short Stories, Smart Cities und die Westchester Counties.</example>
```

Comment:
Instead of a particular exception, wouldn't it be better to add Smart Cities to the lists of words (besides Smart City)?

Reply:
On the one hand, it's true, because in the end all the words that are now ignored by the multitoken rule due to the antipattern are still not accepted by the speller. On the other hand, I created an "ignore_spelling" rule in the disambiguation file to avoid speller matches. I hope that works. Nevertheless, adding them to spelling is a good idea.